client/components/Price/Price.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import styles from './Price.css';
+
+export default class Price extends PureComponent {
+  static propTypes = {
+    price: PropTypes.string,
+    onChange: PropTypes.func,
+  };
+
+  handleOnChangeAction = (e) => {
+    const { price } = this.props;
+    const value = price.length ? price.split(',') : [];
+    const index = value.indexOf(e);
+    if (index > -1) value.splice(index, 1);
+    else value.push(e);
+    this.props.onChange(value.toString());
```

Comment:
you've destructured `this.props` already, why not include `onChange`

Reply:
did some changes here. please check thanks